### PR TITLE
comments form started

### PR DESCRIPTION
**Pull request by Nicolakirk**

### CODE REVIEW

File: wild-swimming-app/src/components/add-comment-form/Addcommentform.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+const [err, setErr] = useState("");
+const[commentsList, setCommentsList]= useState("");
+
+const[message, setMessage]= useState("");
+
+
+const handleSubmit = (event:any)=>{
+  event.preventDefault();
+    const inputComment ={
+     name:userName,
```
Comment (amnioc) [inline wild-swimming-app/src/components/add-comment-form/Addcommentform.tsx:26]: great use of camel case, as they won't have usernames but the user does have a Name!

File: wild-swimming-app/src/components/add-comment-form/Addcommentform.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+     body:bodyMessage,
+    };
+setCommentsList((currentComments)=>{
+  return ([inputComment, ...currentComments])})
+                    setMessage("We have updated your comment");
```
Comment (amnioc) [inline wild-swimming-app/src/components/add-comment-form/Addcommentform.tsx:31]: could this be 'added'? as we won't be doing PATCH requests for MVP

File: wild-swimming-app/src/pages/home/Homepage.tsx [hunk 3]
```diff
@@ -1,10 +1,15 @@
 import Header from "../../components/Header/Header";
+import Map from "../../feature/map/Map";
 
 const Homepage = () => {
   return (
     <section>
+
       <Header />
       <p>Homepage</p>
+      <Map/>
```
Comment (amnioc) [inline wild-swimming-app/src/pages/home/Homepage.tsx:10]: I can see there is a conflict with this merge, was Map left in for your local visibility? 
I am happy for this to be added in ahead of project presentation, but just would want that Homepage mmerges aren't removing anything (which it doesn't look like!)

**Review by amnioc — COMMENTED** [review]
There is conflict between current versions, has the postcode merge been tested with the new Map? i'm happy to approve if you can see a resolution to conflict.

File: wild-swimming-app/src/components/add-comment-form/Addcommentform.tsx (hunk 1 above)
Comment (Nicolakirk) [reply wild-swimming-app/src/components/add-comment-form/Addcommentform.tsx:26]: yes, i've not managed to add that from  the log in info but i will update when I do

File: wild-swimming-app/src/components/add-comment-form/Addcommentform.tsx (hunk 2 above)
Comment (Nicolakirk) [reply wild-swimming-app/src/components/add-comment-form/Addcommentform.tsx:31]: yes sure ill change it to added

**Pull request closed without merging**